# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1739147840 -3600
# Parent  9eb64f29d0f228440b868231fc99876079fb9c74
Misc: Don't ever show 'No message found', it adds no value.

diff --git a/mail/base/content/about3Pane.js b/mail/base/content/about3Pane.js
--- a/mail/base/content/about3Pane.js
+++ b/mail/base/content/about3Pane.js
@@ -5053,13 +5053,12 @@ var threadPane = {
         }
         this.scrollDetected = true;
         break;
       case "showplaceholder":
-        threadTree.updatePlaceholders([
-          folderTree.selection.size > 1
-            ? "placeholderMultipleFolders"
-            : "placeholderNoMessages",
-        ]);
+        // Don't ever show 'No message found', it adds no value.
+        threadTree.updatePlaceholders(
+          folderTree.selection.size > 1 ? ["placeholderMultipleFolders"] : []
+        );
         break;
     }
   },
   observe(subject, topic, data) {
